scene/2d/camera_2d.cpp

```diff
@@ -54,19 +54,22 @@ void Camera2D::_update_scroll() {
 	if (is_current()) {
 		ERR_FAIL_COND(custom_viewport && !ObjectDB::get_instance(custom_viewport_id));
 
+		Size2 screen_size = _get_camera_screen_size();
+		Point2 screen_offset = (anchor_mode == ANCHOR_MODE_DRAG_CENTER ? (screen_size * 0.5) : Point2());
+		Point2 adj_screen_pos;
+
 		Transform2D xform;
 		if (is_physics_interpolated_and_enabled()) {
 			xform = _interpolation_data.xform_prev.interpolate_with(_interpolation_data.xform_curr, Engine::get_singleton()->get_physics_interpolation_fraction());
+			// The following alternative to xform.affine_inverse().get_origin() is more efficient.
+			adj_screen_pos = xform.basis_xform(-xform.get_origin());
```

Comment:
Should we be calculating interpolated `camera_screen_center` here? It looks like this function is just bound for users, but it could have been something I missed originally. Then the code for the adjusted center could be the same.

But maybe this interpolation is calculated after the user `_process()` in which case it wouldn't be so useful (I can't remember the order of operations here off hand).